internal/controlplane/handlers_authz.go

```diff
@@ -54,27 +47,12 @@ func isSuperadmin(claims auth.UserPermissions) bool {
 	return claims.IsStaff
```

Comment:
Since we no longer set `.IsStaff`, should we remove this method as well?

Reply:
We are still using it in some tests to bypass authorization. I created a new issue to remove the superadmin role, since it needs some more refactoring and a better testing appoach. https://github.com/stacklok/minder/issues/2070